tests/acceptance/standard/test_hooks.py

```diff
@@ -111,3 +111,41 @@ def test_hooks_delete(self, gl, project, urls):
         assert first_url not in (h.url for h in hooks)
         assert second_hook in hooks
         assert second_hook == orig_second_hook
+
+    def test_hooks_enforce(self, gl, project, urls):
+        target = project.path_with_namespace
+        first_url, second_url = urls
+        gl_hooks = project.hooks.list()
+        assert len(gl_hooks) == 1
+        assert second_url == gl_hooks[0].url
+
+        enforce_yaml = f"""
+                projects_and_groups:
+                  {target}:
+                    hooks:
+                      enforce: true
+                      {first_url}:
+                        merge_requests_events: false
+                        note_events: true
+                """
+
+        run_gitlabform(enforce_yaml, target)
+        gl_hook_urls = [h.url for h in project.hooks.list()]
```

Comment:
What do you think of naming the hooks list more expressive. For example: `hooks_before_test` , `hooks_after_test`. The code might be more readable that way.